scripts/vtk_convert.py

```diff
@@ -0,0 +1,91 @@
+from vtk import vtkPolyDataReader, vtkStructuredPointsReader, \
+    vtkXMLImageDataWriter, vtkXMLPolyDataWriter
+from girder_client import GirderClient
+import os
+import click
+import re
+
+@click.command()
+@click.option('--inputfolderid', required=True, prompt=True, help='Source Folder ID')
+@click.option('--outputfolderid', required=True, prompt=True, help='Destination Folder ID')
+@click.option('--apikey', required=True, prompt=True, help='API Key from Girder')
+@click.option('--hostname', default='https://data.computational-biology.org/api/v1/', prompt=True, help='URL, make sure to include /api/v1/ at the end')
+def main(inputfolderid, outputfolderid, apikey, hostname):
+  gc = GirderClient(apiUrl=hostname)
+  gc.authenticate(apiKey=apikey)
+
+  def getFiles(itemData):
+    print('getting ' + itemData['name'])
+    dataFiles = gc.get('item/' + itemData['_id'] + '/files')
```

Comment:
If you want, I'm happy to require this script to require Python > 3.6, which would allow us to use [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings). Otherwise, the typical Python practice is to use [string formatting](https://pyformat.info/) instead of concatenation.

Reply:
I myself don't have Python 3.6 on this computer. I'll install it.

Reply:
Ok ends up I had it but it wasn't the default. Just fixed that.